### ISSUE TITLE
colors are inverted 

### ISSUE THREAD

**Issue by MahmoudMabrok** [PatilShreyas/Capturable#193]
I found after capture colors are inverted, i.e. white became black and so on 

see attachment 

![Screenshot_1722091789](https://github.com/user-attachments/assets/dca9fc7b-355b-4d29-81bf-f598be895a6a)

snippet: 

                Image(
                    painter = painterResource(barCode),
                    modifier = Modifier
                        .fillMaxWidth()
                        .height(100.dp)
                        .capturable(captureController),
                    contentDescription = null
                )

**Comment by PatilShreyas** [comment]
Can you try setting white background before capturable modifier?

**Comment by MahmoudMabrok** [comment]
I found issue is only for background, content is being captured properly, 

I think issue comes from part of asAndroidBitmap because bitmap before this part is captured fine

**Comment by abd3lraouf** [comment]
PixelCopy internally ?

**Comment by qaz5823091** [comment]
Try to set background attribute to the last, it may be worked!

I met a similar problem that my background is turned to black.
<img src="https://github.com/user-attachments/assets/6c119cb4-06e4-4f5f-82e7-09f716ed68e9" width="250">
and my code is like this:
```kotlin
Box(
        modifier = Modifier
            .padding(top = 20.dp, start = 32.dp, end = 32.dp)
            .aspectRatio(boxRatio)
            .background(backgroundColor)
            .capturable(captureController)
) {
...
}
```

To solve this problem, I found another solution using NEW VERSION of Compose UI: [Compose to Bitmap](https://developer.android.com/develop/ui/compose/graphics/draw/modifiers#composable-to-bitmap)
and I find the code which is set background property lastly:
```kotlin
val coroutineScope = rememberCoroutineScope()
val graphicsLayer = rememberGraphicsLayer()
Box(
    modifier = Modifier
        .drawWithContent {
            // call record to capture the content in the graphics layer
            graphicsLayer.record {
                // draw the contents of the composable into the graphics layer
                this@drawWithContent.drawContent()
            }
            // draw the graphics layer on the visible canvas
            drawLayer(graphicsLayer)
        }
        .clickable {
            coroutineScope.launch {
                val bitmap = graphicsLayer.toImageBitmap()
                // do something with the newly acquired bitmap
            }
        }
        .background(Color.White)   // THE LAST
) {
    Text("Hello Android", fontSize = 26.sp)
}
```
So I set background which is below to capturable, it is turned to normal!
<img src="https://github.com/user-attachments/assets/eee81e15-ccc4-4df7-9898-2028a5620180" width="250">
Part of my final code:
```kotlin
Box(
    modifier = Modifier
        .padding(top = 20.dp, start = 32.dp, end = 32.dp)
        .aspectRatio(boxRatio)
        .capturable(captureController)
        .background(backgroundColor)
) {
...
}

...

Button(onClick = {
    coroutineScope.launch {
        try {
            val bitmapAsync = captureController.captureAsync().await()
            val bitmap = bitmapAsync.asAndroidBitmap()
            val bytes = ByteArrayOutputStream()
            bitmap.compress(Bitmap.CompressFormat.JPEG, 100, bytes)
            val path = MediaStore.Images.Media.insertImage(context.contentResolver, bitmap, "Title", null)
            val sendIntent = Intent(Intent.ACTION_SEND).apply {
                putExtra(Intent.EXTRA_STREAM, Uri.parse(path))
                type = "images/JPEG"
            }
            val shareIntent = Intent.createChooser(sendIntent, null)
            startActivity(context, shareIntent, null)
        } catch (error: Throwable) {
            // Error occurred, do something.
        }
    }
})
```

Hope to this solution is helpful for you!

**Comment by PatilShreyas** [comment]
Thanks @qaz5823091 for sharing your learnings.

**Comment by PatilShreyas** [comment]
Probably fixed in `v3.0.0`, please try and let us know if the issue re-occurs.
